docs/release-notes/2.36.0.md

```diff
@@ -19,14 +19,20 @@ Percona Monitoring and Management (PMM) is an open source database monitoring, m
 
 <!---
 
-### Release Highlights1
+### pg_stat_monitor 2.0 support in QAN
```

Comment:
@capri1989 @JiriCtvrtka could you please review this part? Thanks

Reply:
LGTM